File: java/program/Dpr.java
// Kelas DPR
public class Dpr{
    //Atribut
    private String id;
    private String name;
    private String bidang;
    private String partai;

    //constructor
    public Dpr(){
        this.id = "";
        this.name = "";
        this.bidang = "";
        this.partai = "";
    }
    
    //constructor
    public Dpr(String id, String name, String bidang, String partai){
        this.name = id;
        this.name = name;
        this.name = bidang;
        this.name = partai;
    }
    
    //Setter dan Getter
    public String getId(){
        return this.id;
    }

    public void setId(String id){
        this.id = id;
    }

    public String getName(){
        return this.name;
    }

    public void setName(String name){
        this.name = name;
    }

    public String getBidang(){
        return this.bidang;
    }

    public void setBidang(String bidang){
        this.bidang = bidang;
    }

    public String getPartai(){
        return this.partai;
    }

    public void setPartai(String partai){
        this.partai = partai;
    }

}

File: Python/program/Dpr.py
# Kelas DPR
class Dpr:
    # Atribut
    __id = ""
    __name = ""
    __bidang = ""
    __partai = ""

    #constructor
    def __init__(self, id = "", name = "", bidang = "", partai = ""):
        self.__name = name
        self.__id = id
        self.__bidang = bidang
        self.__partai = partai

    #getter dan setter
    def get_name(self):
        return self.__name

    def set_name(self, name):
        self.__name = name

    def get_id(self):
        return self.__id
    
    def set_id(self, id):
        self.__id = id
  
    def get_bidang(self):
        return self.__bidang
    
    def set_bidang(self, bidang):
        self.__bidang = bidang

    def get_partai(self):
        return self.__partai
    
    def set_partai(self, partai):
        self.__partai = partai